util.js

```diff
@@ -717,3 +717,10 @@ exports.callbackify = callbackify;
 const textEncoding = require('text-encoding-polyfill');
 exports.TextEncoder = textEncoding.TextEncoder;
 exports.TextDecoder = textEncoding.TextDecoder;
+
+Object.defineProperty(global, 'TextEncoder', {
```

Comment:
Expose the `text-encoding-polyfill` above to the global object to replicate the same behavior as[ node v11.0.0](https://nodejs.org/api/util.html#class-utiltextencoder).